app/src/main/java/io/apicurio/registry/metrics/RestMetricsResponseFilter.java

```diff
@@ -0,0 +1,61 @@
+package io.apicurio.registry.metrics;
+
+import static io.apicurio.registry.metrics.MetricIDs.*;
+
+import static org.eclipse.microprofile.metrics.MetricRegistry.Type.APPLICATION;
+import static org.eclipse.microprofile.metrics.MetricType.COUNTER;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.ext.Provider;
+
+import org.eclipse.microprofile.metrics.Counter;
+import org.eclipse.microprofile.metrics.Metadata;
+import org.eclipse.microprofile.metrics.MetricRegistry;
+import org.eclipse.microprofile.metrics.Tag;
+import org.eclipse.microprofile.metrics.annotation.RegistryType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Provider
+@RestMetricsResponseFilteredNameBinding
+public class RestMetricsResponseFilter implements ContainerResponseFilter {
+
+	@Inject
+	@RegistryType(type = APPLICATION)
+	MetricRegistry metricRegistry;
+
+	private static final Logger log = LoggerFactory.getLogger(RestMetricsResponseFilter.class);
+
+	String REST_RESPONSE_STATUS_CODE_COUNT = "rest_response_status_code";
+	String REST_RESPONSE_STATUS_CODE_COUNT_DESC = "Total number of REST HTTP Response Codes across all endpoints.";
+
+	@Override
+	public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext)
+			throws IOException {
+		log.info("'{} {}'", requestContext.getMethod(), requestContext.getUriInfo().getRequestUri().toString());
```

Comment:
This is info now but I think we should change it to debug or just removing it. Leaving it here for the moment until more iteration is done to the PR

Reply:
Agreed - should be either debug or trace level.

Reply:
After thinking about this more, let's delete this. :) I think it is redundant since we have logging enabled (via annotation) for all the REST implementation classes.

Reply:
:+1: